src/DevLicenseCore.sol

```diff
@@ -324,7 +324,24 @@ contract DevLicenseCore is Initializable, AccessControlUpgradeable, IDevLicenseD
 
         uint256 timestampInit = $._signers[tokenId][signer];
         uint256 timestampCurrent = block.timestamp;
```

Comment:
nitpick: Do we think this variable is adding much clarity?

Reply:
probably not for someone who actually programs contracts 😂 
Question does this version add extra gas since I am adding a variable to the stack or would it get compiled out?